### PR TITLE
CA-381075 Fix the Copyright statement for XenCert source code

### PR DESCRIPTION
**Pull request by catherineliuchina**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,7 +1,7 @@
                   GNU LESSER GENERAL PUBLIC LICENSE
                        Version 2.1, February 1999
 
- Copyright (C) 1991, 1999 Free Software Foundation, Inc.
+ Copyright (c) 2023 Cloud Software Group, Inc.
```
Comment (DeliZhangX) [inline LICENSE:4]: This is GPL license v2.1, should not change it.

File: Makefile [hunk 2]
```diff
@@ -7,8 +7,8 @@ INSTALL_BIN := install -D -m755
 
 # Julian Chesterfield, July 2007
 #
-# Copyright (c) 2007 XenSource, Inc. All use and distribution of this
-# copyrighted material is governed by and subject to terms and
+# Copyright (c) 2023 Cloud Software Group, Inc. All use and distribution
```
Comment (DeliZhangX) [inline Makefile:10]: Similar to discussed, this is history info, need not to change/update, you can add new item.

File: src/XenCert/StorageHandler.py [hunk 3]
```diff
@@ -1,5 +1,4 @@
-# Copyright (c) 2005-2022 Citrix Systems Inc.
```
Comment (DeliZhangX) [inline src/XenCert/StorageHandler.py:1]: These are similar, need not to remove as my understanding.

Comment (catherineliuchina) [reply src/XenCert/StorageHandler.py:1]: I have recovered it

**Review by DeliZhangX — APPROVED** [review]

**Review by stephenchengCloud — APPROVED** [review]

**Pull request merged**